src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -68,16 +68,16 @@ public static Dataset datasetRequestToDataset(DatasetRequestModel datasetRequest
     final List<? extends StorageResource<?, ?>> storageResources =
         cloudPlatform.createStorageResourceValues(datasetRequest);
 
-    boolean secureMonitoringEnabled =
-        Objects.requireNonNullElse(datasetRequest.isSecureMonitoringEnabled(), false);
+    boolean enableSecureMonitoring =
+        Objects.requireNonNullElse(datasetRequest.isEnableSecureMonitoring(), false);
```

Comment:
I know the convention is to prefix boolean properties with `is`, but I think `enableSecureMonitoring` already implies a boolean value

Reply:
I agree, but this name is auto-generated by OpenAPI

Reply:
Oh okay! I didn't realize that